### DISCUSSION TITLE
Ability to define other unit conversion systems

### DISCUSSION THREAD

**Discussion by simoami** [josdejong/mathjs#3183] [Ideas]
Hello, I'm using mathjs for a medical application which uses Conventional Units over SI (e.g. Celsius for temperature over Kelvin and mmHg for blood pressure, instead of Pa, Bar or kg / (m s^2)). See example Lab unit tables:
https://www.labcorp.com/resource/si-unit-conversion-table
https://connect.springerpub.com/content/book/978-0-8261-0976-7/back-matter/app2
https://www.cmeinfo.com/wp-content/uploads/2021/09/0.-Laboratory-Units-Conversion-Table.pdf

Is there a way to customize the behavior for another system? For example, allow users to define a series of methods similar to `.toSI()`, such as `toImperial()`, `.toMetric()`, `.toMedical()`...etc.

**Comment by josdejong** [comment]
Thanks for your suggestion. It would be nice indeed to have more control over the type of unit system used to simplify the units. This should be addressed by [UnitMath](https://github.com/ericman314/unitmath), which we want to integrate in mathjs (replacing the build in `Unit` class), see the docs of UnitMath about `simplify`, for example:

```js
unit('8 kg m / s^2').simplify({ system: 'si' })
```

See:

- https://github.com/josdejong/mathjs/issues/1282
- https://github.com/josdejong/mathjs/discussions/2738
